doc/engineering-reference/src/building-system-simulation-system-manager/steam-systems-and-component-models.tex

```diff
@@ -88,31 +88,31 @@ \subsubsection{Steam Generation at Saturated Conditions:}\label{steam-generation
 
 Case 1: Loop Operating at Saturation Conditions (no superheat), calculating enthalpy of saturated steam at boiler outlet temperature.
 
-Boiler Outlet Temperature = 105BC,
+Boiler Outlet Temperature = 105\(^{\circ}\)C,
 
 \begin{equation}
-{h_{Case1:NoSuperheat}} = [{h_f}_{{{105}^ \bullet }C}]
+{h_{Case1:NoSuperheat}} = [{h_f}_{{{105}^ \circ }C}]
 \end{equation}
 
 \begin{equation}
 {h_{Case1:NoSuperheat}} = \,\,\,\,2684000\,\,\,\,\,\,J
```

Comment:
This stuff definitely could have been cleaned out also.  It's not a priority, but just something to address as you move forward.

Reply:
@Myoldmopar I went to a "standard" of using \circ.  I found that it was the best option.  The "degree symbol" from Word did NOT always process correctly so I went with what seemed to work.  Some places there might have been a lower-case o instead of \circ.  I went with \circ based on what I saw on-line.  At least with almost everything the same now if we don't like this we can do a search and replace if necessary.

Reply:
Oh I wasn't talking about the degree symbol.  I was referring to the `\,\,\,\,` stuff.

Reply:
@Myoldmopar Oic.  Yes, those are annoying.  I have been trying to eliminate those as I see them but I'm not certain I got them all because if an equation looks correct in PDF I don't always scrutinize it in tex.  I guess at some point a search on "\," should help get rid of them though I suppose right now they aren't hurting anything?